addons/base_import/models/base_import.py

```diff
@@ -969,9 +1266,9 @@ def do(self, fields, columns, options, dryrun=False):
             # pad front as data doesn't contain anythig for skipped lines
             r = import_result['name'] = [''] * skipped
             # only add names for the window being imported
-            r.extend(x[index_of_name] for x in data[:import_limit])
+            r.extend(x[index_of_name] for x in input_file_data[:import_limit])
             # pad back (though that's probably not useful)
-            r.extend([''] * (len(data) - (import_limit or 0)))
+            r.extend([''] * (len(input_file_data) - (import_limit or 0)))
```

Comment:
Are the renames really critical? They add further noise to the diff.

Reply:
Actually when reading the import and trying to understand a bit the process (without beers), we proposed some renaming of "data" and "fields" variables to try to better differentiate them. Sorry. That's probably the manque of comté.